cmd/track/track_bundle_test.go

```diff
@@ -204,6 +205,19 @@ func Test_TrackBundleCommand(t *testing.T) {
 			expectPrune:        true,
 			expectInEffectDays: 666,
 		},
+		{
+			name: "verify backward compatibility of in-effect-days flag",
+			args: []string{
+				"--bundle",
+				"registry/image:tag",
+				"--in-effect-days",
+				"100",
+			},
+			expectUrls:         []string{"registry/image:tag"},
+			expectStdout:       true,
+			expectPrune:        true,
+			expectInEffectDays: 100,
+		},
```

Comment:
I don't understand what this test adds that isn't tested in the previous (line 196) test case. Remove it maybe?